languages/wikipron/README.md

```diff
@@ -2,24 +2,26 @@
 ====================
 
```

Comment:
minor generic comment throughout: `TSV` not `tsv` (at least in comments---snake_case if it's a variable name, sure.)